src/test/scala/DiceBotTest.scala

```diff
@@ -0,0 +1,14 @@
+import org.scalatest.FunSuite
+
+class DiceBotTest extends FunSuite {
+  Seq(
+    "invalid string",
+    null,
+    "").foreach(x => {
+    test(s"process with invalid input '$x' returns none.") {
+      val expected = None
+      val actual = DiceBot.process("invalid string")
+      assert(expected == actual)
```

Comment:
The way you write the test code is somewhat different with the test codes I've seen on the Exercism. What is advantage of using  ```Seq ... foreach ... ``` thing? I have no issue with the way you writing it. Just want to know. :)